public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -62,153 +77,161 @@ code-example(language="bash").
+  +makeExample('toh-6/ts/app/hero.service.ts', 'to-promise')(format=".")
+  :marked
+    Unfortunately, the Angular `Observable` doesn't have a `toPromise` operator ... not out of the box.
+    The Angular `Observable` is a bare-bones implementation.
 
-  There are scores of operators like `toPromise` that extend `Observable` with useful capabilities.
-  If we want those capabilities, we have to add the operators ourselves.
-  That's as easy as importing them from the RxJS library like this:
-+makeExample('toh-6/ts/app/hero.service.ts', 'rxjs')(format=".")
+    There are scores of operators like `toPromise` that extend `Observable` with useful capabilities.
+    If we want those capabilities, we have to add the operators ourselves.
+    That's as easy as importing them from the RxJS library like this:
+  +makeExample('toh-6/ts/app/hero.service.ts', 'rxjs')(format=".")
+
+  :marked
+    ### Extracting the data in the *then* callback
+    In the *promise*'s `then` callback we call the `json` method of the http `Response` to extract the
+    data within the response.
+  +makeExample('toh-6/ts/app/hero.service.ts', 'to-data')(format=".")
 
 :marked
-  ### Extracting the data in the *then* callback
-  In the *promise*'s `then` callback we call the `json` method of the http `Response` to extract the
-  data within the response.
-+makeExample('toh-6/ts/app/hero.service.ts', 'to-data')(format=".")
-:marked
-  That object returned by `json` has a single `data` property.
-  The `data` property holds the array of *heroes* that the caller really wants.
-  So we grab that array and return it as the resolved promise value.
+  That response JSON has a single `data` property.
+  The `data` property holds the !{_array} of *heroes* that the caller really wants.
+  So we grab that !{_array} and return it as the resolved !{_Promise} value.
 
 .alert.is-important
   :marked
     Pay close attention to the shape of the data returned by the server.
-    This particular *in-memory web api* example happens to return an object with a `data` property.
-    Your api might return something else.
+    This particular *in-memory web API* example happens to return an object with a `data` property.
+    Your API might return something else.
 
-    Adjust the code to match *your web api*.
+    Adjust the code to match *your web API*.
 :marked
-  The caller is unaware of these machinations. It receives a promise of *heroes* just as it did before.
-  It has no idea that we fetched the heroes from the server.
-  It knows nothing of the twists and turns required to turn the http response into heroes.
+  The caller is unaware of these machinations. It receives a !{_Promise} of *heroes* just as it did before.
+  It has no idea that we fetched the heroes from the (mock) server.
+  It knows nothing of the twists and turns required to convert the HTTP response into heroes.
```

Comment:
Personally I don't like either `turn` nor `convert`. I would personally say `map` which is a programming concept too.